docs/contributing.rst

```diff
@@ -42,3 +42,27 @@ Pull requests should be submitted early and often!
 If your pull request is not yet ready to be merged, please also include the **[WIP]** prefix.
 This tells the development team that your pull request is a "work-in-progress",
 and that you plan to continue working on it.
+
+Release Checklist
+`````````````````
+
+This is the checklist of items that must be completed when cutting a new release of tedana.
+These steps can only be completed by a project maintainer, but they a good resource for
+releasing your own Python projects!
+
+    #. All continuous integration must be passing and docs must be building successfully
+    #. Create a new release, using the GitHub `guide for creating a release on GitHub`_.
+       `Release-drafter`_ should have already drafted release notes listing all
+       changes since the last release; check to make sure these are correct.
+    #. Pulling from the ``master`` branch, locally build a new copy of tedana and
+        `upload it to PyPi`_.
+
+We have set up tedana so that releases automatically mint a new DOI with Zenodo;
+a guide for doing this integration is available `here`_.
+You can also `learn more about DueCredit`_ and why citing software is important.
```

Comment:
This kind of comes out of nowhere now, given that the duecredit citation for the package no longer needs to be updated manually.